src/GraphQl/Mutation/MutationResolverInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\GraphQl\Mutation;
+
+use GraphQL\Type\Definition\ResolveInfo;
+
+/**
+ * Interface MutationInterface.
+ *
+ * @experimental
+ *
+ * @author Raoul Clais <raoul.clais@gmail.com>
+ */
+interface MutationResolverInterface
+{
+    public function __invoke($item, array $input, ResolveInfo $info);
```

Comment:
Shouldn't the mutation return something? How is it take into account by the Schema generator?